Automattic/jetpack
# Add Welcome banner to My Jetpack

## Proposed changes:

* This PR adds a new banner to My Jetpack as a Welcome message to Jetpack users. This banner has a close button and is dismissed (after refresh) if the user interacts with any CTA on the page, so the banner is not annoying.

<img width="1520" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/31260786-4812-4025-82ba-8399dbc3f235">

### Other information:


## Jetpack product discussion


p1HpG7-pPp-p2#comment-67823

## Does this pull request change what data or activity we track or use?


It adds a few more events when the user views and/or clicks to dismiss the banner

## Testing instructions:






* Spin up a new JN site pointing Jetpack Beta to this branch
* Visit My Jetpack and make sure the banner is there
* If you click to close the banner, it will close and won't show again after you refresh the page
* If you click on the main CTAs (Add, Manage, Purchase, Fix), the banner won't show again after you refresh the page (it'll be there if you just go back to the previous page)
